Assets/Scripts/Score/ScoreManager.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using TMPro;
+
+public class ScoreManager : MonoBehaviour
+{
+    public static ScoreManager Instance { get; private set; }
+    public float Score
+    {
+        get { return score; }
+        private set { score = value; }
+    }
+
+    [SerializeField]
+    private TextMeshProUGUI scoreText;
+
+    private const float moveMulti4Pts = 10.0f, // The value to multiply the distance traveled by
+                        enemyFelledPts = 100.0f; // The points killing an enemy is worth
```

Comment:
I think that these values should be set in the Inspector (probably also `transitionSpeed`). Then, there should be a Score Manager prefab with an instance in each level.

Reply:
It's good for configuration values to be settable in the Inspector. Then, code familiarity is not needed for changing those values. In this case, it doesn't matter that much.